examples/source/1.components/amp-story-interactive-poll/index.html

```diff
@@ -0,0 +1,200 @@
+              option-2-text="Brushing mustache" option-2-results-category="Mouse" option-2-confetti="👺"
+              option-3-text="Jumping" option-3-results-category="Bunny" option-3-confetti="🦘"
+              option-4-text="Fetching balls" option-4-results-category="Dog" option-4-confetti="⚽">
+            </amp-story-interactive-poll>
+          </div>
+        </amp-story-grid-layer>
+      </amp-story-page>
+
+      <!-- And let's make a third one just to train muscle memory
+      
+      There's multiple attributes and CSS variables exposed for creators to adapt the style to fit the story design.
+      Adding gradients to the prompt-background is a great way to enhance the visuals of your elements, for instance.
+      
+      `--interactive-prompt-background: linear-gradient(120deg, var(--color-1), var(--color-2));`
+
+      [tip type="note"]If you want to see all the available styles, check the [documentation](/documentation/components/amp-story-interactive?format=stories) or [example story](/documentation/examples/components/amp-story-interactive/story).[/tip]
+      
+      -->
+      <amp-story-page id="page-poll-3">
+        <amp-story-grid-layer template="fill">
+          <amp-img
+            layout="fixed" width="1600" height="1200"
+            src="https://images.unsplash.com/photo-1500835556837-99ac94a94552?q=20"/>
+        </amp-story-grid-layer>
+        <amp-story-grid-layer template="fill" aspect-ratio="4:6">
+          <div animate-in="scale-fade-up">
+            <amp-story-interactive-poll
+               id="poll-3"
+               prompt-text="What would you do for holidays?"
+               endpoint="https://amp.dev/documentation/examples/components/amp-story-interactive-poll/results"
+               class="center poll-3"
+               prompt-size="large"
+               chip-style="shadow"
+               option-1-text="Easter island" option-1-results-category="Bunny" option-1-confetti="🗿"
+               option-2-text="Drive with tongue out" option-2-results-category="Dog" option-2-confetti="🏎️"
+               option-3-text="Run from cats" option-3-results-category="Mouse" option-3-confetti="🏃"
+               option-4-text="Stay in bed" option-4-results-category="Cat" option-4-confetti="😴">
+            </amp-story-interactive-poll>
+          </div>
+        </amp-story-grid-layer>
+      </amp-story-page>
+
+      <!-- 
+        ## Adding the results
+        We have specified `option-{1/2/3/4}-results-category` on all the polls without any immediate effect, but here is where they come into play.
+
+        When users select the options in the polls previously created, an internal state saves those options and checks what `results-category` each option selected had. This element reads the state from all the polls, and counts how many selected options correspond to each of the states configured in the amp-story-interactive-results. Then it shows the category, description and image of the calculated state.
```

Comment:
I tried to write a suggestion update, but in this paragraph at "This element reads the state from all the polls..." I got confused what "This element" is referring to - is it the entire `amp-story-interactive` component?

Reply:
It refers to the `amp-story-interactive-results` element created below. The hint is that the section is named "adding the results" but what do you suggest we add here to make it more clear?